etc/httpd/global.conf

```diff
@@ -760,3 +760,6 @@ RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?firefox/beta/feedbackprivacypolicy/?)$ /b
 
 # bug 1149019
 RewriteRule ^(/certs/.*)$ /b$1 [PT]
+
+# bug 1252332
+RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?sync/?)$ /b$1 [PT]
```

Comment:
This is what I was doing when converting old redirects to passthroughs just because it was easier :) Either way should be fine. This file is no longer in use in stage and prod, but will continue to affect demos and dev. I'll try soon to reduce this file to a single passthrough for the whole site and we can forget about it forever :fireworks: